### PR TITLE
Upgrade awscli to 1.7.12

### PR DESCRIPTION
**Pull request by radeksimko**

### CODE REVIEW

File: Library/Formula/awscli.rb [hunk 1]
```diff
@@ -96,6 +101,9 @@ def install
   end
 
   def caveats; <<-EOS.undent
+    Python >2.7.2 is recommended to prevent urllib3 SSL warnings:
```
Comment (adamv) [inline Library/Formula/awscli.rb]: I'm -1 on adding this message, but don't feel that strongly about it if other maintainers don't.

Comment (mikemcquaid) [inline Library/Formula/awscli.rb]: I agree.

Comment (radeksimko) [inline Library/Formula/awscli.rb]: The only reason is that each time the user will call any `aws` command that talks to the API (which is pretty much any one) will see the warning and may think that awscli is broken. But I'm happy with removing it.

Comment (tdsmith) [inline Library/Formula/awscli.rb]: This is only relevant on 10.7, which ships 2.7.1. We could upgrade the conditional on `depends_on :python` to `<= :lion`.

File: Library/Formula/awscli.rb [hunk 2]
```diff
@@ -69,8 +69,13 @@ class Awscli < Formula
...
   resource "rsa" do
-    url "https://pypi.python.org/packages/source/r/rsa/rsa-3.1.2.tar.gz"
-    sha1 "ebf54ad3fff8bc1df09f5d777d5a913e5aef8df5"
+    url "https://pypi.python.org/packages/source/r/rsa/rsa-3.1.4.tar.gz"
+    sha1 "208583c49489b7ab415a4455eae7618b7055feca"
+  end
+
+  resource "wsgiref" do
+    url "https://pypi.python.org/packages/source/w/wsgiref/wsgiref-0.1.2.zip"
+    sha1 "80b7e9b039e40a2f8419e00b393a6516d80cf8de"
```
Comment (tdsmith) [inline Library/Formula/awscli.rb]: What requires wsgiref?

Comment (tdsmith) [inline Library/Formula/awscli.rb]: Or, rather, wsgiref was added to the stdlib in 2.5 and this formula is guaranteed to be run with 2.7 so I think it's unnecessary.

**Pull request closed without merging**